docs/coding-guidelines/package-projects.md

```diff
@@ -56,17 +56,17 @@ Sample `System.Collections.Concurrent.pkgproj`
 ```
 
 ### Framework-specific library
-Framework specific libraries are effectively the same as the previous example.  The difference is that the src project reference **must** refer to the `.builds` file which will provide multiple assets from multiple projects.
+Framework specific libraries are effectively the same as the previous example.  The difference is that the src project reference **must** refer to the `.proj` file which will provide multiple assets from multiple projects.
```

Comment:
This is way out of date.  This is how we did things in 1.x.  We should just delete this section as it isn't meaningfully different now that we handle this in the project itself.